### ISSUE TITLE
🚀 Feature:  Project global environment

### ISSUE THREAD

**Issue by mbos2** [appwrite/appwrite#5316]
### 🔖 Feature description

**Global project environment(s)**

Environment page for environment variables per project.
I can be placed inside project settings. 

Those environment variables would be accessible via `process.env` or somehow from the appwrite package.
example

```
import appwrite from 'appwrite';
const env = appwrite.env;
// cont apiKey = env.API_KEY;
```

Api secret keys could be added automatically into some hidden environment that could also be called from appwrite package.
example
```
import appwrite from 'appwrite';
const env = appwrite.env.api;
// cont testApiKey= env.Test;
```

Maybe process.env might not be good option because there could be colision with other, more local, env files. 

Possibility of more customization, like:
- Env variables that are only accessible in functions
- Env variables that are only accessible in specific services
- - Per env variable

### 🎤 Pitch

In my use case I would use API keys from different services. 
Several functions could benefit from api key defined only once, instead in each of the functions separately.
Furthermore it would not be exposed. 

It could also reduce necessity of creating .env files.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by mbos2** [comment]
> Hi, I think we have a related issue here: #3530

Well, it is only partly related. 

Global project environment would be accessible through the whole project, which includes functions too. 

Functions only environments means it is accessible only via functions. 

So please don't close this, it's not duplicate of https://github.com/appwrite/appwrite/issues/3530 :)

**Comment by Mushmou** [comment]
Hi I would like to work on this issue.

**Comment by joeyouss** [comment]
Hi @mbos2 , we would like to tell you this issue is being worked on by our team and will be available soon : )

**Comment by stnguyen90** [comment]
@mbos2, 

> Global project environment would be accessible through the whole project, which includes functions too.

I'm not sure if this is a good idea since things like API keys should be secret and shouldn't be included client-side.

As for having some config value accessible via the SDK, that sounds like https://github.com/appwrite/appwrite/issues/323. What do you think?

**Comment by mbos2** [comment]
Remote config seems interesting feature indeed.

**Comment by eldadfux** [comment]
![Image](https://github.com/user-attachments/assets/48fcfaff-4370-4e47-a0c2-a02986df2dce)
Global env vars at project level have been released in Appwrite 1.4 and are available on all your functions runtimes. 🎉
